Ruby/Embeddings/SQL (for Ruby).sublime-syntax

```diff
@@ -12,3 +12,7 @@ contexts:
     - meta_prepend: true
     - include: Ruby.sublime-syntax#escaped-char
     - include: Ruby.sublime-syntax#embedded-ruby
+
+  comments:
+    - include: Ruby.sublime-syntax#escaped-char
+    - include: Ruby.sublime-syntax#embedded-ruby
```

Comment:
Are you sure about this one? 

`comments` is included in `prototype` and replacing all SQL comments may not have intentional effects.

Maybe we should think about removing `- meta_include_prototype: false` from comment contexts in SQL (Basic) instead, so `prototype` and thus ruby extensions are applied normally. 

Alternatively prepend those rules to
- inside-double-dash-comment
- inside-comment-docblock
- inside-comment-block

Reply:
You're absolutely right that there is a mistake here - I thought I had done a `meta_append`...

I like your alternative suggestions. I will be able to take a look at fixing it tomorrow night. If anyone is impatient and want to work on it before then, be my guest 🙂